pyafq_cli.json

```diff
@@ -0,0 +1,501 @@
+        "default-value": "None",
+        "optional": true
+    },
+    {
+        "id": "seed_mask",
+        "name": "seed mask",
+        "type": "String",
+        "value-key": "[SEED_MASK]",
+        "default-value": "None",
+        "optional": true
+    },
+    {
+        "id": "n_seeds",
+        "name": "n seeds",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[N_SEEDS]",
+        "minimum": 1,
+        "default-value": 1,
+        "optional": true
+    },
+    {
+        "id": "random_seeds",
+        "name": "random seeds",
+        "type": "String",
+        "value-key": "[RANDOM_SEEDS]",
+        "value-choices": ["false", "true"],
+        "default-value": "false",
+        "optional": true
+    },
+    {
+        "id": "stop_mask",
+        "name": "stop mask",
+        "type": "String",
+        "value-key": "[STOP_MASK]",
+        "default-value": "None",
+        "optional": true
+    },
+    {
+        "id": "stop_threshold",
+        "name": "stop threshold",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[STOP_THRESHOLD]",
+        "minimum": 0,
+        "default-value": 0,
+        "optional": true
+    },
+    {
+        "id": "step_size",
+        "name": "step size",
+        "type": "Number",
+        "value-key": "[STEP_SIZE]",
+        "minimum": 0.0,
+        "default-value": 0.5,
+        "optional": true
+    },
+    {
+        "id": "min_length",
+        "name": "min length",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[MIN_LENGTH]",
+        "minimum": 1,
+        "default-value": 10,
+        "optional": true
+    },
+    {
+        "id": "max_length",
+        "name": "max length",
+        "type": "Number",
+        "integer": true,
+        "value-key": "[MAX_LENGTH]",
+        "default-value": 1000,
+        "optional": true
+    },
+    {
+        "id": "odf_model",
+        "name": "odf model",
+        "type": "String",
+        "value-key": "[ODF_MODEL]",
+        "value-choices": ["DTI"],
+        "default-value": "DTI",
+        "optional": true
+    },
+    {
+        "id": "wm_labels",
+        "name": "wm labels",
+        "type": "Number",
+        "integer": true,
+        "list": true,
+        "min-list-entries": 1,
+        "list-separator": ", ",
+        "value-key": "[WM_LABELS]",
+        "value-choices": [250, 251, 252, 253, 254, 255, 41, 2, 16, 77],
+        "default-value": [250, 251, 252, 253, 254, 255, 41, 2, 16, 77],
+        "optional": true
+    },
+    {
+        "id": "nb_points",
+        "name": "nb points",
+        "type": "String",
+        "value-key": "[NB_POINTS]",
+        "value-choices": ["false", "true"],
+        "default-value": "false",
+        "optional": true
+    },
+    {
+        "id": "segmentation_seg_algo",
+        "name": "segmentation seg algo",
+        "type": "String",
+        "value-key": "[SEGMENTATION_SEG_ALGO]",
+        "value-choices": ["AFQ"],
```

Comment:
Similar to `bundles_seg_algo`, does `segmentation_seg_algo` only have `AFQ` and `Reco` as the valid options?

Reply:
Yes.